tests/procfs/procfs.c

```diff
@@ -99,13 +99,26 @@ int test_cpuinfo()
     close(fd);
 }
 
+int test_fdatasync()
+{
+    int fd;
+    char buf[1024];
+
+    fd = open("/proc/cpuinfo", O_RDONLY);
+    assert(fd > 0);
+    int ret = fdatasync(fd);
+    assert(ret == -1 && errno == EINVAL);
+    close(fd);
+}
+
```

Comment:
We should have a test case for hostfs. 
1. stat a host file 
2. Open the host file
3. write to it
4. call fdatasync
5. do stat again, ensure file size has changed.

Reply:
Added test case on these lines in `/tests/fs`.